app/templates/leaderboard.html

```diff
@@ -1,14 +1,23 @@
 ﻿{% block style %}
-<link rel="stylesheet" href="../static/content/leaderboardstyle.css">
+<link rel="stylesheet" href="{{ url_for('static', filename='content/leaderboardstyle.css') }}">
 {% endblock %}
 {% block content %}
 {% include "sidebar.html" %}
 <div class="leaderboard">
     <header class="leaderboard-heading">
         <h1 class="leaderboard-title"><span>Class Leaderboard</span></h1>
     </header>
-
+    
     <main class="leaderboard-table">
+        <!--{% for user in users %}
+        <article class="leaderboard-row">
+            <h1>{{loop.index}}.</h1>
+            <img src="https://randomuser.me/api/portraits/men/32.jpg" alt="{{ user.firstName }}" class="leaderboard-img">
+            <span class="leaderboard-name">{{ user.firstName }}</span>
```

Comment:
should probably have `{{ user.firstName }} {{ user.lastName }}` just in case got people with same first name